packages/template-tachyons/src/Css.ts

```diff
@@ -710,6 +717,17 @@ export type Padding = "padding" | "paddingTop" | "paddingRight" | "paddingBottom
 
 type Brand<K, T> = K & { __brand: T };
 type Breakpoint = Brand<string, "Breakpoint">;
+export type BreakpointKey = "print" | "sm" | "md" | "smOrMd" | "mdAndUp" | "mdAndDown" | "lg" | "mdOrLg";
+export enum Breakpoints {
+  print = "@media print",
+  sm = "@media screen and (max-width:599px)",
+  md = "@media screen and (min-width:600px) and (max-width:959px)",
+  smOrMd = "@media screen and (max-width:959px)",
+  mdAndUp = "@media screen and (min-width:600px)",
+  mdAndDown = "@media screen and (max-width:959px)",
+  lg = "@media screen and (min-width:960px)",
+  mdOrLg = "@media screen and (min-width:600px)",
+}
```

Comment:
let me know if we should change `BreakpointKey` to something else - reusing the term `Breakpoint(s)` a lot

Reply:
:thinking: yeah good point...

We do have breakpoints-the-`const`, breakpoints-the-type-union, and breakpoints-the-enum...

Wondering if we need all three of these? 

I think the current setup generally assumed the users would use consts, like:

```
import { sm } from "src/Css.ts"
<div css={{ [sm]: { color: Css.red.$ }} />
```

It looks like you're adding these types so that we could do something like:

```
const ... = useBreakpoint("sm");
```

And then strongly type the `"sm"` in `useBreakpoint`?

I kinda wondering if we should double down on the same `const`-based approach, so it'd be like:

```
import { sm } from "src/Css.ts";
const ... = useBreakpoint(sm);
```

And so instead of introducing new type union+enum versions of `sm`, we'd just export the existing `type Breakpoint`?

Then `useBreakpoint` could be declared with like `function useBreakpoint(bp: Breakpoint)` and used like `useBreakpoint(sm)` or what not...?

I'm not dogmatic about the current const approach; I don't remember why we necessarily choose under, other than maybe just slavish chasing the brevity of utility css, i.e. doing `[sm]` is two chars (!!) shorter than `["sm"]` (type union) and much shorter than `[Breakpoints.sm]` (enum).

So, dunno, wdyt?

Reply:
...I'm thinking maybe there was a type reason we did the current approach, like looking at:

```
<div css={{ [sm]: ... }}>...</div>
```

A good question is, "why use the symbol at all?", i.e.:

```
<div css={{ sm: ... }}>...</div>
```

...but, oh right, currently we're not telling emotion (via config / something) that "sm" is the media query "@media ...", it just wants the media query string passed directly as is, so that's why we did the const `sm` approach.

Just had to remind myself...

Reply:
yeah in an attempt to handle the senario where we would need access to multiple conditional break points and talking with @bdow we felt that a record of the breakpoints and bool values would make sense, then the user would get all the potential breakpoints listed via `breakpoints.md` etc

Reply:
Per your mention of wanting a map, I think the enum makes sense. Using the `keyof typeof` makes me feel slightly less bad about having "three ways of exporting 'something called breakpoint'" but each of the use cases makes sense, so sgtm!